roles/openshift_common/tasks/main.yml

```diff
@@ -4,6 +4,9 @@
   hostname: name={{ openshift_bind_ip }}
   when: openshift_hostname_workaround
 
+- name: Update all packages
+  yum: name=* state=latest
+
```

Comment:
Do we really want to update all the packages to the latest here?

Possibly have this run conditionally based on a variable so that we only do it the first time we run through (or if we are intending to do a system update)?